base/BUILD

```diff
@@ -45,6 +45,8 @@ intellij_plugin_library(
         "intellij-ue-2019.3": [],
         "intellij-2020.1": ["sdkcompat/v201/META-INF/blaze-base-v201.xml"],
         "intellij-ue-2020.1": ["sdkcompat/v201/META-INF/blaze-base-v201.xml"],
+        "intellij-2020.2": ["sdkcompat/v201/META-INF/blaze-base-v201.xml"],
+        "intellij-ue-2020.2": ["sdkcompat/v201/META-INF/blaze-base-v201.xml"],
```

Comment:
I think you have to create a new `sdkcompat/v202` directory for the changes. While it's not necessary since we don't care about backwards compatibility, it can help us figure out when we can drop our commits.